content/en/docs/contribute/suggesting-improvements.md

```diff
@@ -10,22 +10,23 @@ card:
 
 {{% capture overview %}}
 
-If you notice an issue with Kubernetes documentation, or have an idea for new content, then file an issue. All you need is a [GitHub account](https://github.com/join) and a web browser.
+If you notice an issue with Kubernetes documentation, or have an idea for new content, then open an issue. All you need is a [GitHub account](https://github.com/join) and a web browser.
 
-All new work on Kubernetes documentation begins with an issue in GitHub. Kubernetes contributors
-then review, categorize and tag issues as needed. You or another member
-of the Kubernetes community begin work on the issue an open a pull request with changes to resolve the issue.
+In most cases, new work on Kubernetes documentation begins with an issue in GitHub. Kubernetes contributors
+then review, categorize and tag issues as needed. Next, you or another member
+of the Kubernetes community open a pull request with changes to resolve the issue.
 
 {{% /capture %}}
 
 {{% capture body %}}
 
-## Suggesting improvements to existing content
+## Opening an issue
+
+If you want to suggest improvements to existing content, or notice an error, then open an issue.
 
 1. Go to the bottom of the page and click the **Create an Issue** button. This redirects you
  to a GitHub issue page pre-populated with some headers.
-2. Using Markdown, fill in as many details as you can. At a minimum, describe what
-  the issue or suggestion for improvement is.
+2. Describe what the issue or suggestion for improvement is. Provide as many details as you can.
```

Comment:
nit: line 29, wording seems a bit off.
Possible edit, something like:
Complete the issue template providing as many details as possible.
Line 46, reword, `To help others understand your issue and can act on it, do the following:`
To help others understand your issue
Line 48, seems to be the same as line 29.